app/controllers/v0/messages_controller.rb

```diff
@@ -61,6 +51,22 @@ def thread
              meta: resource.metadata
     end
 
+    def reply
+      message = Message.new(message_params)
+
+      if message.body.blank?
+        message.errors.add(:body, "can't be blank")
+        raise Common::Exceptions::ValidationErrors, message
+      end
+
+      resource = client.post_create_message_reply(params[:id], message_params)
+      raise VA::API::Common::Exceptions::RecordNotFound, params[:id] unless resource.present?
```

Comment:
This is probably unnecessary since MHV will have raised a different error before this would ever get called.